backend/src/main/java/com/senior/project/backend/Activity/MilestoneRepository.java

```diff
@@ -16,36 +18,79 @@ public class MilestoneRepository {
                 .id("Task 1 id")
-                .description("Task 1 description")
+                .description("Meet with academic advisor to discuss current major and class schedule")
                 .needsArtifact(true)
                 .isRequired(false)
                 .build();
         Task task2 = Task.builder()
                 .id("Task 2 id")
-                .description("Task 2 description")
+                .description("Complete Degreeworks Training")
                 .needsArtifact(true)
                 .isRequired(true)
                 .build();
         List<Task> tasks = new ArrayList<>();
         tasks.add(task1);
         tasks.add(task2);
 
+        Event e1 = Event.builder().eventID("event 1")
+                .name("Major/Minor & Career Exploration Event")
+                .isRecurring(false)
+                .organizer("Organizer 1")
+                .location("Location 1")
+                .isRequired(true).build();
+        e1.setDescription("Event 1 description");
+        e1.setDate(new Date());
+        var events = new ArrayList<Event>();
+        events.add(e1);
+
         Milestone m1 = Milestone.builder()
                 .milestoneID("milestone 1")
-                .name("Milestone 1")
-                .isActive(false)
+                .name("Major Exploration")
+                .isActive(true)
                 .yearLevel(YearLevel.Freshman)
-                .events(EventRepository.DATA)
+                .events(events)
                 .tasks(tasks)
                 .build();
         DATA.add(m1);
+
+
+        Event e2 = Event.builder().eventID("event 1")
+                .name("Attend Job Fair Fall Semester")
+                .isRecurring(false)
+                .organizer("Organizer 1")
+                .location("Location 1")
+                .isRequired(true).build();
+        e1.setDescription("Event 1 description");
+        e1.setDate(new Date());
+        Event e3 = Event.builder().eventID("event 1")
+                .name("Attend Job Fair Spring Semester")
+                .isRecurring(false)
+                .organizer("Organizer 1")
+                .location("Location 1")
+                .isRequired(true).build();
+        e1.setDescription("Event 1 description");
+        e1.setDate(new Date());
+        var events2 = new ArrayList<Event>();
+        events2.add(e2);
+        events2.add(e3);
+
+        Task task3 = Task.builder()
+                .id("Task 3 id")
+                .description("Meet with academic advisor to discuss class schedule and receive PIN for registration")
+                .needsArtifact(true)
+                .isRequired(true)
+                .build();
+        List<Task> tasks2 = new ArrayList<>();
+        tasks2.add(task3);
+
+
         Milestone m2 = Milestone.builder()
                 .milestoneID("milestone 2")
-                .name("Milestone 2")
+                .name("Major/Minor Exploration")
                 .isActive(false)
```

Comment:
what does it mean by active? like is this assigned to the student or something? would a milestone ever be inactive?

Reply:
We had talked about this originally with the sponsor and we added it to the design doc, I was just following that originally, but based on the current understanding of the requirements this probably isn't needed anymore. I'll remove